packages/spaces/commands/vpn/info.js

```diff
@@ -49,6 +50,10 @@ function * run (context, heroku) {
 
   let lib = require('../../lib/vpn-connections')(heroku)
   let info = yield lib.getVPNConnection(space, name)
+
+  if (info.name != null) {
```

Comment:
From my testing, name is an empty string when it's not present.

Reply:
You should be able to just use `if (!info.name) {`:

```
> foo = ''
''
> if (!foo) {
... console.log("oops")
... }
oops
undefined
> foo = null
null
> if (!foo) {
... console.log("oops")
... }
oops
undefined
> foo = 'baz'
'baz'
> if (!foo) {
... console.log("oops")
... }
undefined
```

Reply:
`if (info.name && info.name.length)` ?

Reply:
I don't think that will work in this context because `info.name.length` could return 0 if it's an empty string. Going that route it would have to do an `info.name.length > 0` check, no? Is the more verbose method here preferred over something like a `!info.name` check?

Reply:
`(!info.name)` is good! Empty strings and zero's are falsey.

Reply:
For fun, javascript ~silliness~ madness(?):
```
> "" || 1
1
> 0 || 1
1
> [] || 1
[]
> var a = {};  a || 1
{}
```